rust/kcl-lib/src/execution/cad_op.rs

```diff
@@ -233,7 +233,7 @@ impl From<&KclValue> for OpKclValue {
                 ty: ty.clone(),
             },
             KclValue::String { value, .. } => Self::String { value: value.clone() },
-            KclValue::Array { value, .. } => {
+            KclValue::MixedArray { value, .. } => {
                 let value = value.iter().map(Self::from).collect();
                 Self::Array { value }
             }
```

Comment:
Is the distinction between mixed and homogeneous purely an implementation detail? I'm asking because Operations expose the structure of KclValues. I'm wondering if they should get renamed also or if the two will both output the same `OpKclValue::Array`.

Reply:
I think initially it's just an implementation detail. But pretty soon it will be an apparent difference in types, so it might be useful to distinguish in the ops too

Reply:
Okay. You can merge this as is. We can update it later when those other changes come.